site/docs/v1/tech/apis/system.adoc

```diff
@@ -121,6 +121,11 @@ The format string used to format the date and time columns in the export result.
 +
 When this parameter is omitted a default format of `M/d/yyyy hh:mm:ss a z` will be used. See the https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html[DateTimeFormatter patterns] for additional examples.
 
+[field]#includeArchived# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
```

Comment:
this entire section should be replace by a call to 

```
include::docs/v1/tech/apis/_system-logs-request-body.adoc[]
```

I think. I just checked em out and they appear to be the same content.